frontend/packages/ux-editor/src/components/Properties/Text.tsx

```diff
@@ -64,10 +63,9 @@ export const Text = () => {
                   ComponentSpecificConfig<ComponentType.RadioButtons>)
           }
           handleComponentChange={handleComponentChange}
-          editFormId={formId}
           layoutName={selectedFormLayoutName}
           renderOptions={{
-            onlyCodeListOptions: schema.properties.optionsId && !schema.properties.options,
+            isLayoutOptionsUnsupported: schema.properties.optionsId && !schema.properties.options,
```

Comment:
Kan vi kalle denne `areLayoutOptionsSupported` og gi den motsatt verdi, så vi unngår doble negativer?

Jeg lurer for øvrig på om ikke dette er noe vi kan sette på et lavere nivå? Det blir i så fall en egen PR å rydde opp i det tenker jeg.

Reply:
Fikset. Kan se på å flytte den ned senere.